### PR TITLE
storybook - i18n toolbar

### PR DESCRIPTION
**Pull request by kumavis**
made a toolbar dropdown for selecting a language in storybook i18n

### CODE REVIEW

**Comment by kumavis** [comment]
wait is `addParameters` the same as this toolbar thing?

File: .storybook/locales.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import * as am from '../app/_locales/am/messages.json';
```
Comment (Gudahtt) [inline .storybook/locales.js:1]: TIL that you can't `export * as ___ from ...` yet. [Soon hopefully](https://github.com/tc39/proposal-export-ns-from)!

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Comment by kumavis** [comment]
btw this required a retry of e2e-firefox and benchmark

**Pull request merged**